Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -120,6 +124,7 @@
     <string name="wiimote_scanning_description">Leave this on if you are using a DolphinBar for real Wii Remote support.</string>
     <string name="wiimote_speaker">Wii Remote Speaker</string>
     <string name="wiimote_speaker_description">Enable sound output through the speaker on a real Wii Remote (DolphinBar required).</string>
+    <string name="dualshockudp_client_description">DSU protocol enables the use of input and motion data from compatible sources, like PlayStation, Nintendo Switch and Steam controllers.</string>
```

Comment:
I understand that this description is taken from DolphinQt, but the focus on particular types of controllers doesn't make a lot of sense in this context, mainly because time has passed and all of these controller types should now be working (well, minus motion controls for Switch controllers on Android). What do you think about a string that shifts the focus along the lines of this:

`The DSU protocol lets you send inputs from a separate app or device.`

Reply:
Okay, let's leave it as it is for this PR.